modules/serverless-testing-quarkus-functions.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies
+//
+// * /serverless/functions/serverless-developing-quarkus-functions.adoc
+
+[id="serverless-testing-quarkus-functions_{context}"]
+= Testing Quarkus functions
+
+You can test Quarkus functions locally on your computer by running the Maven tests that are included in the project template.
+
+.Procedure
+
+* Run the Maven tests:
++
+[source,terminal]
+----
+$ ./mvnw test
+----
```

Comment:
Is it possible to include some sort of sample output here? Or maybe it wouldn't actually help in this case? Just for consideration.

Reply:
@lance wdyt? We'd need to add this for all the tests I think if we do it for this one and idk if it makes sense to do that.
I think for now it's OK and we could add them later if you think it makes sense to add more detail, but it's up to you.

Reply:
Test output from that command will typically be several screens long, so I think if we do have some sample output, it would be truncated. For now, I agree it's probably OK as it is. It should be clear to any Quarkus developer if that command succeeds or fails for any reason.